website/components/Integrations.tsx

```diff
@@ -1,6 +1,7 @@
 import { INTEGRATIONS } from "./integrationsData";
 
 const LogoContainer: React.FC<{
+  children?: React.ReactNode;
```

Comment:
This type was removed in newer versions of React to force devs to explicitly set it when needed